source/plugins/ruby/KubernetesApiClient.rb

```diff
@@ -517,7 +517,7 @@ def getContainerResourceRequestsAndLimits(pod, metricCategory, metricNameToColle
               #No container level limit for the given metric, so default to node level limit
             else
               if (metricCategory == "limits" && !nodeAllocatableRecord.nil? && !nodeAllocatableRecord.empty? && nodeAllocatableRecord.has_key?(metricNameToCollect))
-                metricValue = nodeAllocatableRecord[metricNameToCollect]
+                metricValue = getMetricNumericValue(metricNameToCollect, nodeAllocatableRecord[metricNameToCollect])
```

Comment:
Thanks for investigating and making the fix. lets get the image tag update from "ciprod06272022" to "ciprod07282022" in omsagent.yaml and DockerFiles.